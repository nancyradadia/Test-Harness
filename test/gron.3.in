{
  "nested": {
    "key1": "value1",
    "key2": {
      "subkey1": [1, 2, 3],
      "subkey2": "nested_value"
    }
  }
}
