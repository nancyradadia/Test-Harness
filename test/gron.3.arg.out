json = {};
json.nested = {};
json.nested.key1 = "value1";
json.nested.key2 = {};
json.nested.key2.subkey1 = [];
json.nested.key2.subkey1[0] = 1;
json.nested.key2.subkey1[1] = 2;
json.nested.key2.subkey1[2] = 3;
json.nested.key2.subkey2 = "nested_value";